packages/remix-dev/compiler/shims/cloudflare-pagesEntryModule.ts

```diff
@@ -0,0 +1,12 @@
+import { createPagesFunctionHandler } from "@remix-run/cloudflare-pages";
```

Comment:
Let's have a subfolder at `compiler/shims/entryModules` so we don't have mixed-case filenames.